deploy/deploy_test.go

```diff
@@ -24,38 +25,52 @@ import (
 	fakecorev1 "k8s.io/client-go/kubernetes/typed/core/v1/fake"
 
 	ktest "k8s.io/client-go/testing"
+	"k8s.io/klog/v2"
 )
 
+const verbose = true
```

Comment:
what is the point of having this as a const? all of the tests use the true value to print their logs

Reply:
This should be false normally.  Setting it to true prints out the additional debugging information when working on test (i.e., prints what commands were actually executed).